advanced_functionality/multi_model_linear_learner_home_value/linear_learner_multi_model_endpoint_inf_pipeline.ipynb

```diff
@@ -0,0 +1,1302 @@
+{
```

Comment:
<p>A cell above prints the output of <code>create_role</code> call, while the output of <code>create_policy</code> is not printed here. It would be good to have consistency.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

Reply:
<p>removed print after the create_roe call, since we are already printing the role_arn</p>